src/django_components/__init__.py

```diff
@@ -19,6 +19,10 @@
     registry as registry,
 )
 from django_components.library import TagProtectedError as TagProtectedError
+from django_components.slots import (
+    SlotContent as SlotContent,
+    SlotFunc as SlotFunc,
```

Comment:
Renamed `SlotRenderContent` and `SlotRenderFunc` and made them public. [See how SlotFunc is used here](https://github.com/EmilStenstrom/django-components/pull/594/files#diff-b335630551682c19a781afebcf4d07bf978fb1f8ac04c6bf87428ed5106870f5R628)